pages/10.Components/03.Forms/02.Progressive Profiling/docs.en.md

```diff
@@ -17,7 +17,19 @@ There are two possible ways that you can configure a form field to display only
 
 It's recommended to use the email field in each form. From Mautic 2.9 email can be hidden, but be aware of that. Email as identifier of a contact could be unusable if same PC is used by more people (public library, schools...).
```

Comment:
```suggestion
It is recommended to use the Email field in each Form. From [Mautic 2.9][mautic-2.9] the email field can be hidden, but use this with caution. Email as identifier of a Contact could be unusable if same PC is used by more people (public library, schools...).
```

Reply:
Please add the[ link macro at the bottom of the page](https://docs.mautic.org/en/home/contributing#linking-to-release-versions):

`[release-2.9.0]: <https://github.com/mautic/mautic/releases/tag/2.9.0>`